.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - id: set-matrix
+        run: |
+          JSON=$(cat ./composer.json)
+          echo "::set-output name=${JSON//'%'/'%25'}"
+
+  # The actual TER upload job
+  ter-release:
+    needs: set-matrix
+    name: "TYPO3 TER release"
+
+    if: startsWith(github.ref, 'refs/tags/')
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-versions: [ 7.4 ]
+        composer-data: ${{fromJson(needs.set-matrix.outputs.matrix)}}
+
+    env:
+      TYPO3_API_TOKEN: ${{ secrets.TYPO3_API_TOKEN }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Check tag
+        run: |
+          if ! [[ ${{ github.ref }} =~ ^refs/tags/[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$ ]]; then
+            exit 1
+          fi
+
+      - name: Get version
+        id: get-version
+        run: echo ::set-output name=version::${GITHUB_REF/refs\/tags\//}
+
+      - uses: jungwinter/split@v1
+        id: split
+        with:
+          msg: ${{ matrix.composer-data.name}}
+          separator: '/'
+
+      - name: Remove files
+        run: >
+          rm -rf CODE_OF_CONDUCT.md Tests codeception.yml phpcs.xml .github .gitlab .editor .Build .gitattributes
```

Comment:
With the [exclude list included in the latest tailor version](https://github.com/TYPO3/tailor/blob/main/conf/ExcludeFromPackaging.php), we only need to remove the following file (due to a [typo in the exclude list](https://github.com/TYPO3/tailor/pull/19)):
```
.php_cs.php
```